src/api/database/bulk-api.rst

```diff
@@ -456,6 +456,139 @@ Sending multiple queries to a database
     The multiple queries are also supported in /db/_local_docs/queries and
     /db/_design_docs/queries (similar to /db/_all_docs/queries).
 
+.. _api/db/bulk_get:
+
+==================
+``/db/_bulk_get``
+==================
+
+.. http:post:: /{db}/_bulk_get
+    :synopsis: Fetches several documents at the given revision
+
+    This method is used by PouchDB and other client to improve the pull
+    replications, by allowing the client to request multiple documents in one
+    request.
```

Comment:
This is nice info, but the context is wrong. The tone is somewhat out of place as compared to the other endpoints; can you rewrite this to discuss the endpoint without passing value judgments on other endpoints?

Reply:
I'm not sure: is it about this sentence or about the text just below?